deps/spidershim/test/template.cc

```diff
@@ -441,6 +438,7 @@ static void TestSignature(V8Engine& engine, Local<Context> context,
   }
 }
 
+#if 0
```

Comment:
Yes.  That test also requires `SetAccessorProperty` before it can pass, which I'm implementing on top of this...  I will eventually remove the `#if 0` once that patch is ready.